status-reporter/status_reporter/status_reporter.py

```diff
@@ -152,8 +151,8 @@ def _collect_status(self):
 
     def _report(self):
         status, services = self._collect_status()
-        if not isinstance(status, dict):
-            logger.error('Ignoring status: expected a report in dict format,'
+        if not isinstance(services, dict):
```

Comment:
in Yoni's PR there is the range validation for status?

Reply:
Yes Yoni will update it in his PR